packages/bcd.rb

```diff
@@ -0,0 +1,99 @@
+require 'package'
+
+class Bcd < Package
```

Comment:
New package?  Can we put this in a separate PR?

Reply:
That's leftover, I forgot to get it in c9bf791, I'll take care of it in the next commit.